azure-pipelines/build.yml

```diff
@@ -49,6 +56,22 @@ steps:
     codeCoverageTool: 'Cobertura'
     summaryFileLocation: '$(Agent.TempDirectory)/coverlet/reports/Cobertura.xml'
 
+- task: PowerShell@2
+  displayName: 'Generate Nuspec Version'
+  inputs:
+    targetType: 'inline'
+    script: |
+      $propsFile = '$(Build.SourcesDirectory)\Directory.Build.Props'
+      $props = New-Object XML
+      $props.Load($propsFile)
```

Comment:
Is this just a complicated way to do:

$props = [xml](Get-Content -Raw $propsFile)

?

Reply:
I had some trouble accessing properties with Get-Content, but maybe casting to XML would have helped. 
I think it's fine to use this unless there's a risk with xml.load